src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/CanvasView.cs

```diff
@@ -89,6 +89,29 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// The viewbox of the CanvasView.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Size2D ViewBox
```

Comment:
Hello, could you please check this first? 
https://code.sec.samsung.net/jira/browse/GRE-2025
we currently recommend NOT to use class type as a property. 
so I think it would be good to change `Size2D ViewBow` property to other type.
This is just an example :
public int ViewBoxSizeWidth, public int ViewBoxSizeHeight, public SetViewBoxSize(Size2D size), and etc.
please check again. thank you.

Reply:
I updated the patch. Replaced by SetViewBoxSize/GetViewBoxSize methods.